src/backer/backering.py

```diff
@@ -376,13 +380,14 @@ def __next__(self):
 class MailboxIterable:
     TimeoutMBX = 30000000
 
-    def __init__(self, mbx, pre, topics, retry=5000, hab=None, ledger=None):
+    def __init__(self, mbx, pre, topics, retry=5000, hab=None, ledger=None, queue=None):
```

Comment:
`queue` shouldn't default to `None` as we need it. Also, I think we can remove `ledger` completely, and just check `if topic == "/receipt"`